src/components/dialogs/network-modifications/line/creation/line-creation-dialog.js

```diff
@@ -144,7 +144,11 @@ const LineCreationDialog = ({
         resolver: yupResolver(formSchema),
     });
 
-    const { reset, setValue } = formMethods;
+    const {
+        reset,
+        setValue,
+        formState: { errors, isSubmitting },
```

Comment:
be careful if you get values from react hook form in root component, it will rerender the whole dialog each time one of those values changes